MongoDBService/MongoDBService/Source/Core/Collection.swift

```diff
@@ -40,7 +41,7 @@ public struct Collection {
     @discardableResult
     public func find(query: Document,
                      projection: Document? = nil,
-                     limit: Int) -> StitchTask<[Document]> {
+                     limit: Int) -> Promise<[Document]> {
```

Comment:
So we're gonna need another major change right?

Reply:
Correct